pkg/apis/agones/v1/apihooksfake.go

```diff
@@ -56,9 +56,9 @@ func (f fakeAPIHooks) MutateGameServerPodSpec(gss *GameServerSpec, podSpec *core
 }
 
 // SetEviction is called by gs.Pod to enforce GameServer.Status.Eviction.
-func (f fakeAPIHooks) SetEviction(safe EvictionSafe, pod *corev1.Pod) error {
+func (f fakeAPIHooks) SetEviction(eviction *Eviction, pod *corev1.Pod) error {
```

Comment:
@zmerlynn I made this change to handle Eviction being a pointer. PTAL and let me know what you think of the approach.

Reply:
LGTM